### DISCUSSION TITLE
Migrate to a fresh installation

### DISCUSSION THREAD

**Discussion by Nebelfuchs** [pretix/pretix#4523] [Q&A] [answered]
Hi all,
is there a somewhat easy way to migrate a pretix postgres db dump to a fresh installation?

**Comment by raphaelm** [comment]
It's possible, but ideally also you have the media files: https://docs.pretix.eu/en/latest/admin/maintainance.html

**Reply by Nebelfuchs** [reply]
Thanks! I am thinking about migrating from our current docker setup to a manual installation. I guess I'd just have to copy the db dump and data directory to a new VM and start from there with this https://docs.pretix.eu/en/latest/admin/installation/manual_smallscale.html ?

**Reply by raphaelm** [reply, marked as answer]
Yes
